styles.css

```diff
@@ -0,0 +1,939 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+:root {
+  --clr-purple: #6070ff;
+  --clr-white: #fff;
+  --clr-para: #344563;
+  --clr-gray-100: #b3bac5;
+}
+
+* {
+  margin: 0;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  background-color: #e5e5e5;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.logo a,
+.menu-icon a {
+  color: var(--clr-purple);
+  font-weight: 700;
+  font-size: 18px;
+}
+
+.navigation {
+  display: flex;
+  justify-content: space-between;
+  padding: 15px;
+  background-color: var(--clr-white);
+  position: fixed;
+  left: 0;
+  right: 0;
+  z-index: 1;
+  align-items: center;
+}
+
+.dev a {
+  font-weight: 500;
+  color: #7f8cff;
+}
+
+.nav-link a {
+  color: #fff;
+}
+
+.social-icons a {
+  margin-right: 1.2rem;
+}
+
+.nav-link a:hover {
+  color: var(--clr-para);
+}
+
+.bar1 {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: var(--clr-purple);
+}
+
+.bar2 {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: var(--clr-purple);
+}
+
+.bar3 {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: var(--clr-purple);
+}
+
+.menu-icon.active .bar1 {
+  transform: translateY(8px) rotate(45deg);
+  background: #fff;
+}
+
+.menu-icon.active .bar3 {
+  transform: translateY(-8px) rotate(-45deg);
+  background: #fff;
+}
```

Comment:
- [ ] Wow, a nice way to create the close (X) button :heart_eyes: . But to follow [HTML & CSS requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) let's put our code this way:
```sh
.menu-icon.active > span:nth-child(odd) {
  background: #fff;
}

.menu-icon.active .bar1 {
  transform: translateY(8px) rotate(45deg);
}

.menu-icon.active .bar3 {
  transform: translateY(-8px) rotate(45deg);
}
```
So we do not need to repeat the `background: #fff` property. You can read more about CSS :nth-child() selector [here](https://www.w3schools.com/cssref/sel_nth-child.php).